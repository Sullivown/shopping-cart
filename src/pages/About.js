import React, { useEffect } from 'react';
import updateDocumentTitle from '../helpers/updateDocumentTitle';
import HappyCloud from '../assets/images/happy-cloud.png';

function About(props) {
	useEffect(() => {
		updateDocumentTitle(props.title);
	});

	return (
		<div className='page-content about'>
			<h2>About Us</h2>
			<p>
				Our Story started in 1789 when our founder, Sir Brantham Dante,
				started selling his wares on the corner of Moth Lane and Cricks
				Street. Over the years his shop evolved into an adventuring
				equipment specialist and was sold to its current owner in 1851.
				All our adventuring items are hand tested, cleaned and often
				refurbished to make sure items reach you in the best possible
				condition. We sell a wide range of goods including: Weapons,
				armour, potions, adventuring equipment and many more.
			</p>
			<img src={HappyCloud} alt='A happy cloud!' />
		</div>
	);
}

export default About;
