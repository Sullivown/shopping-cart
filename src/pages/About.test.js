import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import About from './About';

describe('Test About component', () => {
	it('renders', () => {
		const AboutTestRenderer = TestRenderer.create(<About />).toJSON();
		expect(AboutTestRenderer).toMatchSnapshot();
	});
});
