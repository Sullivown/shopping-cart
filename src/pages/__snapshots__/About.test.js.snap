// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test About component renders 1`] = `
<div
  className="page-content about"
>
  <h2>
    About Us
  </h2>
  <p>
    Our Story started in 1789 when our founder, Sir Brantham Dante, started selling his wares on the corner of Moth Lane and Cricks Street. Over the years his shop evolved into an adventuring equipment specialist and was sold to its current owner in 1851. All our adventuring items are hand tested, cleaned and often refurbished to make sure items reach you in the best possible condition. We sell a wide range of goods including: Weapons, armour, potions, adventuring equipment and many more.
  </p>
  <img
    alt="A happy cloud!"
    src="happy-cloud.png"
  />
</div>
`;
